mne_bids/dig.py

```diff
@@ -182,29 +225,233 @@ def _coordsystem_json(raw, unit, orient, coordsystem_name, fname,
                          'same coordinate frame. Found: "{}"'
                          .format(coord_frame))
 
+    # get the coordinate frame description
+    coordsystem_desc = COORD_FRAME_DESCRIPTIONS.get(coordsystem_name, "n/a")
+
+    # create the coordinate json data structure based on 'kind'
```

Comment:
I think it'd be good to include a log message here (not a warning, just a message that can be handled with verbose)

it should say that if your coordsystem_name is `Other` ... you HAVE TO manually define the CoordsystemDescription in your dataset later on.

Because the whole purpose of BIDS is defeated it people put "other" somewhere and never describe what it is.
```suggestion
    if coordsystem_name == 'Other' and verbose:
        print('Using the `Other` keyword for the CoordinateSystem field. '
              'Please specify the CoordinateSystemDescription field manually.')
    
    # create the coordinate json data structure based on 'kind'
```